src/solutions/nonlinear_solutions.jl

```diff
@@ -1,3 +1,20 @@
+
+"""
+$(TYPEDEF)
+Statistics from the nonlinear equation solver about the solution process.
+## Fields
+- nf: Number of function evaluations.
+- nsolve: The number of linear solves `W\b` required for the integration.
```

Comment:
split factorizations and backsolves?

Reply:
yeah. not sure if that makes sense, I was just modeling this on the diffeq one. I'm happy to combine these

Reply:
why would that not make sense?

Reply:
I think it does.

Reply:
fixed?